nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -657,6 +657,11 @@ systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
      since Nextcloud doesn't support upgrades across multiple major versions.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The packages perl, rsync and strace where removed from systemPackages. If you need them, install them again with <literal>environment.systemPackages = with pkgs; [ perl rsync strace ];</literal> in your <literal>configuration.nix</literal>.
```

Comment:
```suggestion
      The packages <package>perl</package>, <package>rsync</package> and <package>strace</package> where removed from <option>systemPackages</option>. If you need them, install them again with <code><xref linkend="opt-environment.systemPackages"/> = with pkgs; [ perl rsync strace ];</code> in your <filename>configuration.nix</filename>.
```

Compare [`<code>`](https://tdg.docbook.org/tdg/5.1/code.html#element.db.code.6)  and [`<literal>`](https://tdg.docbook.org/tdg/5.1/literal.html#element.db.literal.6).

Reply:
Thanks. I'm not sure if it's worth the effort when we switch the documentation format and there might just be `<code>`.

Reply:
At least rst (the current most likely candidate) should support that using [roles](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-option): 

```rst
      The packages :package:`perl`, :package:`rsync` and :package:`strace` where removed from :option:`systemPackages`. If you need them, install them again with <code>:option:`environment.systemPackages` = with pkgs; [ perl rsync strace ];</code> in your :file:`configuration.nix`.
```

though I am not sure how to make parsed inline literals for the option within `code`.